dbms/src/Storages/DeltaMerge/DeltaIndex.h

```diff
@@ -145,10 +145,12 @@ class DeltaIndex
 
     String toString()
     {
-        std::stringstream s;
-        s << "{placed rows:" << placed_rows << ", deletes:" << placed_deletes << ", delta tree: " << delta_tree->numEntries() << "|"
-          << delta_tree->numInserts() << "|" << delta_tree->numDeletes() << "}";
-        return s.str();
+        return fmt::format("<placed_rows={} placed_deletes={} tree_entries={} tree_inserts={} tree_deletes={}>",
```

Comment:
Why use "<...>" instead of "{...}"?

Reply:
I discovered that MPP part is using `<` for log prefixes, like:

```
DeltaMergeStore:MPP<query:435777869046874113,task:1> Table<69> [sorted_ranges: 1] [tasks before split: 1] [tasks final: 1] [ranges final: 1]
```

so I made an update as well. I'm fine to either `{}` or `<>` as they are both visually easy to be discovered as a delimiter. What's your opinion? @JaySon-Huang

Reply:
OK. Keep the logging style aligned to existing one is better